src/lib/Components/Inbox/Conversations/Composer.tsx

```diff
@@ -74,6 +80,8 @@ export default class Composer extends React.Component<Props, State> {
       paddingRight: 10,
     }
 
+    const disableSendButton = !(this.state.text && this.state.text.length) || this.props.disabled
```

Comment:
FYI I’d generally revert this to _first_ check the simple property and only if that doesn’t short-circuit the conditional do some computation.